### ISSUE TITLE
Excessive pruning with ranged reactors

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-Py#1386]
I've run into an interesting issue when using the new ranged reactors feature. Essentially, RMG will eventually run the simulations at conditions where the previous chemistry becomes unimportant and then immediately prune almost the entire edge. In this case, the run completely successfully with no species or reactions in the edge.

Some logging statements towards the end of the run:
```
After model enlargement:
    The model core has 394 species and 21749 reactions
    The model edge has 2256 species and 3646 reactions
... (running simulation)
Pruning 1788 species whose rate ratios against characteristic rate did not exceed the minimum threshold of 0.0005
... (running simulation)
Pruning 468 species whose rate ratios against characteristic rate did not exceed the minimum threshold of 0.0005
... (running simulation)
MODEL GENERATION COMPLETED

The final model core has 394 species and 21749 reactions
The final model edge has 0 species and 0 reactions
```

**Comment by mjohnson541** [comment]
So I think the best solution to this is to increase minSpeciesIterationsForPrune to be proportional to max(nSimsTerm for each reactor) * # reactors.  This can be done directly in the current input file or I could internally multiply the given value by those factors if we think that's more convenient.

**Comment by mjohnson541** [comment]
I think I've worked out the problem.  Prior to this we had say six reactors we ran and then we checked maxEdgeSpeciesRateRatios variables on each reactor.  Now that attributes is being overwritten every time run a simulation, which means that regardless of how many cycles you wait in ranged reactors pruning is being done only for the last condition tested in the reactor.

**Comment by mliu49** [comment]
Fixed by #1394.
